sdk/evaluation/azure-ai-evaluation/CHANGELOG.md

```diff
@@ -44,13 +47,23 @@ outputs = asyncio.run(custom_simulator(
 
 ### Other Changes
 - Improved error messages for the `evaluate` API by enhancing the validation of input parameters. This update provides more detailed and actionable error descriptions.
+- `GroundednessEvaluator` now supports `query` as an optional input in single-turn evaluation. If `query` is provided, a different prompt template will be used for the evaluation.
 - To align with our support of a diverse set of models, the following evaluators will now have a new key in their result output without the `gpt_` prefix. To maintain backwards compatibility, the old key with the `gpt_` prefix will still be present in the output; however, it is recommended to use the new key moving forward as the old key will be deprecated in the future.
   - `CoherenceEvaluator`
   - `RelevanceEvaluator`
   - `FluencyEvaluator`
   - `GroundednessEvaluator`
   - `SimilarityEvaluator`
   - `RetrievalEvaluator`
+- The following evaluators will now have a new key in their result output including LLM reasoning behind the score. The new key will follow the pattern "<metric_name>_reason". The reasoning is the result of a more detailed prompt template being used to generate the LLM response. Note that this requires the maximum number of tokens used to run these evaluators to be increased.
+
+| Evaluator | New Token Limit |
+| --- | --- |
+| `CoherenceEvaluator` | 800 |
+| `RelevanceEvaluator` | 800 |
+| `FluencyEvaluator` | 800 |
+| `GroundednessEvaluator` | 800 |
+| `RetrievalEvaluator` | 1600 |
```

Comment:
Could you please add a 2-space indent in front of the table so it aligns under the current bullet ('-') section?

Reply:
Done